### PR TITLE
Annotate test_singleton_lifecycle_fast.py

### PR DESCRIPTION
**Pull request by AmineKhaldi**

### CODE REVIEW

File: tests/wallet/test_singleton_lifecycle_fast.py [hunk 1]
```diff
@@ -31,25 +31,25 @@
...
-    def add_puzzle(self, puzzle: Program):
+    def add_puzzle(self, puzzle: Program) -> None:
         self._db[puzzle.get_tree_hash()] = Program.from_bytes(bytes(puzzle))
 
     def puzzle_for_hash(self, puzzle_hash: bytes32) -> Optional[Program]:
         return self._db.get(puzzle_hash)
 
 
-def from_kwargs(kwargs, key, type_info=Any):
+def from_kwargs(kwargs: Dict[str, Any], key: str, type_info: Any = Any) -> Any:
```
Comment (arvidn) [inline tests/wallet/test_singleton_lifecycle_fast.py:52]: this looks fishy:
```
type_info: Any = Any
```

Should it not be `Type[Any]`?

**Review by arvidn — APPROVED** [review]

File: tests/wallet/test_singleton_lifecycle_fast.py (hunk 1 above)
Comment (AmineKhaldi) [reply tests/wallet/test_singleton_lifecycle_fast.py:52]: I tried that originally but I got `error: Incompatible default for argument "type_info" (default has type "object", argument has type "type[Any]")  [assignment]`. Please note this is currently not even used (as noted in the function's docstring) so I deemed `Any` to be adequate at this point.

Comment (arvidn) [reply tests/wallet/test_singleton_lifecycle_fast.py:52]: could we just delete the parameter instead?

Comment (AmineKhaldi) [reply tests/wallet/test_singleton_lifecycle_fast.py:52]: Sure. PR #17375.

**Pull request merged**